nautobot/core/views/mixins.py

```diff
@@ -25,7 +25,6 @@
 from rest_framework.viewsets import GenericViewSet
 
 from drf_spectacular.utils import extend_schema
-
```

Comment:
Personal preference (non-blocking) would be to restore this line, I like keeping Nautobot imports separate from other package imports.